libs/angular-accelerator/src/lib/components/interactive-data-view/interactive-data-view.component.ts

```diff
@@ -40,7 +51,10 @@ export type InteractiveDataViewComponentState = ColumnGroupSelectionComponentSta
   selector: 'ocx-interactive-data-view',
   templateUrl: './interactive-data-view.component.html',
   styleUrls: ['./interactive-data-view.component.css'],
-  providers: [{ provide: 'InteractiveDataViewComponent', useExisting: InteractiveDataViewComponent }],
+  providers: [
+    { provide: 'InteractiveDataViewComponent', useExisting: InteractiveDataViewComponent },
+    { provide: SLOT_SERVICE, useExisting: SlotService },
```

Comment:
Slot component requires SLOT_SERVICE token. We need to provide it in components or in modules (like here https://github.com/onecx/onecx-user-profile-ui/blob/71328357fc3e58ffb037c00e84bc1761257aeda9/src/app/onecx-user-profile-remote.module.ts#L56). Problem here is that products do not know about the slot because its burried in libs, therefore I would leave it like it is